docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -253,6 +253,59 @@ These parameters can be mapped using the ``expand`` and ``partial`` methods. Not
         trigger_kwargs=[{"moment": timedelta(hours=2)}, {"moment": timedelta(hours=2)}]
     )
 
+Exiting deferred task from Triggers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ .. versionadded:: 2.10.0
+
+If you want to exit your task directly from the triggerer without going into the worker, you can specific operator level attribute ``end_from_trigger`` with the attributes to your deferrable operator other discussed above. This can save some resources needed to start a new worker.
```

Comment:
```suggestion
If you want to exit your task directly from the triggerer without going into the worker, you can specify the instance level attribute ``end_from_trigger`` with the attributes of your deferrable operator, as discussed above. This can save some resources needed to start a new worker.
```

I feel it's not actually operator-level. The example looks like an instance-level attribute